test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -818,6 +899,34 @@ public async Task HttpTrigger_Identities_AnonymousAccessSucceeds()
             }
         }
 
+        [Fact]
+        public async Task HttpTrigger_StaticWebApps_Identities_AnonymousAccessSucceeds()
+        {
+            var vars = new Dictionary<string, string>
+            {
+                { RpcWorkerConstants.FunctionWorkerRuntimeSettingName, RpcWorkerConstants.DotNetLanguageWorkerName},
+                { "STATIC_WEB_APPS_FUNCTION", "TRUE"}
+            };
+            using (_fixture.Host.WebHostServices.CreateScopedEnvironment(vars))
+            {
+                string uri = $"api/httptrigger-identities";
+                HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, uri);
+
+                MockStaticWebAppsClientPrincipal(
+                    request,
+                    "aad",
```

Comment:
nit: If you could specify the parameter names for all of these parameters, it would help with clarity with the sequential strings about which is which.